docs/metodologias-agiles.md

```diff
@@ -29,3 +43,128 @@ Consiste en la elaboración de un cuadro o diagrama en el que se reflejan tres c
 Es una metodología basada en un conjunto de reglas y buenas prácticas para el desarrollo de software en ambientes muy cambiantes con requerimientos imprecisos, por ende, está enfocada en la retroalimentación continua entre el equipo de desarrollo y el cliente. XP tiene como base la simplicidad y como objetivo la satisfacción del cliente.
 
 <iframe width="100%" height="350" src="https://www.youtube.com/embed/h34e1ph0aPk" frameborder="0" allow="true" allowfullscreen></iframe>
+
+
+## Gestión de proyectos ágiles
+
+### Historias de Usuarios
+
+Una historia de usuario es un método ligero para capturar rápidamente el **quién**, **qué** y **por qué** de los requisitos de un producto. En términos simples, las historias de usuario son ideas de requisitos que expresan lo que necesitan los usuarios. Las historias de los usuarios son breves y cada elemento suele contener menos de 10 o 15 palabras cada uno.  
+El propósito de una historia de usuario es articular cómo una pieza de trabajo devolverá un valor particular al cliente. Tenga en cuenta que los **clientes** no tienen que ser usuarios finales externos en el sentido tradicional, también pueden ser clientes internos o colegas dentro de su organización que dependen de su equipo.
+
+<iframe width="100%" height="350" src="https://www.youtube.com/embed/Zi9E1aUO_1U" frameborder="0" allow="true" allowfullscreen></iframe>
+
+#### ¿Cómo escribir una historias de usuario?
+
+- **Definición de Terminado:** La historia generalmente se “termina” cuando el usuario puede completar la tarea descrita.
+- **Resuma las subtareas o tareas:** Decida qué pasos específicos deben completarse y quién es responsable de cada uno de ellos.
+- **Definir usuarios:** ¿Para quién?, Si hay varios usuarios finales, considere crear varias historias.
```

Comment:
I think there is no need for a comma, correct me if I am wrong, everything else is great!

Reply:
Can be left on or can be removed, both are ok, thanks!!